analyzers/src/SonarAnalyzer.Common/DiagnosticAnalyzer/DiagnosticDescriptorBuilder.cs

```diff
@@ -85,14 +67,6 @@ public static class DiagnosticDescriptorBuilder
                 dd.HelpLinkUri,
                 dd.CustomTags.Union(new[] { WellKnownDiagnosticTags.NotConfigurable }).ToArray());
 
-        [Obsolete]  // Will be removed with obsolete GetDescriptor method
-        private static string[] BuildTags(string diagnosticId, ResourceManager resourceManager, bool fadeOutCode) =>
-            BuildTags(
-                resourceManager.GetString("RoslynLanguage"),
-                bool.Parse(resourceManager.GetString($"{diagnosticId}_IsActivatedByDefault")),
-                (SourceScope)Enum.Parse(typeof(SourceScope), resourceManager.GetString($"{diagnosticId}_Scope")),
-                fadeOutCode);
-
         private static string[] BuildTags(string languageName, bool sonarWay, SourceScope scope, bool fadeOutCode)
```

Comment:
Who knows when will be the next time to touch this class, I suggest to change `BuildUtilityTags` to expression bodied method.